tests/ecm_prep_test.py

```diff
@@ -17019,18 +17019,18 @@ def test_mseg_fail_inputs(self):
             with self.assertRaises(Exception):
                 measure.check_meas_inputs()
 
-    def test_mseg_fail_missing(self):
-        """Test 'fill_mkts' function given measure with missing baseline data.
+    # def test_mseg_fail_missing(self):
```

Comment:
Delete or uncomment?

Reply:
@jtlangevin I see that this test doesn't pass as of the commit where it gets commented out. Can we drop (delete) the test or do we need to fix it?

Reply:
Yes, the lines at 4715-4718 in the updated ecm_prep file set the measure cost/performance to baseline in cases where it's missing so that no msegs are silently removed from the analysis. So this test would now be obsolete, and I think can be removed.